### PR TITLE
Upgrade to docker-compose 3.3 and add wait-for script

### PR DESCRIPTION
**Pull request by rauny-brandao**
What do you guys think about these changes?

- Upgrade to docker-compose version 3.3
- Add script to wait for the postgres to be ready, to solve the problem where postgres starts and restarts to create db and roles. [issues/4](https://github.com/Unleash/unleash-docker/issues/4)
- Update README.md with project requirements

Logs:

- docker-compose rm -v:
docker-compose-rm-v

- docker-compose build:
docker-compose-build

- docker-compose up:
docker-compose-up-01

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,6 +6,8 @@ RUN npm install --production
 
 COPY . .
 
+RUN chmod +x ./wait-for
```
Comment (ivarconr) [inline Dockerfile]: Is it possible to avoid having to make it an executable inside the Dockerfile?

E.g.: Could it just be an executable?

**Comment by ivarconr** [comment]
> upgrade to docker-compose version 3.3, it requires Docker engine 17.06.0+

I was actually a bit confused here. docker-compose should be at version 1.16.1, which support compose file format version 3. 

https://github.com/docker/compose/releases/tag/1.16.1

**Comment by ivarconr** [comment]
We should probably document requirements in README.md

File: Dockerfile (hunk 1 above)
Comment (rauny-brandao) [reply Dockerfile]: I've replaced the wait-for script with the following script:

```
command: > 
  sh -c "
    while ! nc -z db 5432; do
      echo 'Postgress is unavailable.'
      sleep 1
    done
    npm run start"
```

What do you think?

**Pull request merged**